### DISCUSSION TITLE
Make fractions in the ingredient list look a bit nicer

### DISCUSSION THREAD

**Discussion by bartkummel** [mealie-recipes/mealie#3724] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

The fractions in the ingredient list don't look so nice, because the spacing is too wide. 
<img width="260" alt="Scherm­afbeelding 2024-06-11 om 16 30 50" src="https://github.com/mealie-recipes/mealie/assets/20771942/8ca0e842-ee79-4acd-ad5d-0e37e26fc1ae">

### Please provide a concise description of the feature that would resolve your issue.

By making the spacing more tight, the fractions will look more naturally, as if a unicode fraction character was used.
<img width="248" alt="Scherm­afbeelding 2024-06-11 om 16 30 59" src="https://github.com/mealie-recipes/mealie/assets/20771942/900ae4ff-b15c-4f80-9f0a-35bddddcb86c">

### Please consider and list out some caveats or tradeoffs made in your design decision

The tradeoff is that we need a bit more CSS to go over the wire.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by bartkummel** [comment]
The situation is a bit different than I thought when I created this request, I'll illustrate with screenshots:
- On version `1.8.0`, it looks like this: <img width="248" alt="1 8 0" src="https://github.com/mealie-recipes/mealie/assets/20771942/8fad5f63-1c70-47b4-bc51-cdf4031b1680">
- On version `next`, it looks like this: <img width="250" alt="next-before" src="https://github.com/mealie-recipes/mealie/assets/20771942/b1c36459-12da-478a-8997-147ac0cdea85">
- With my change applied, it looks like this: <img width="256" alt="next-after" src="https://github.com/mealie-recipes/mealie/assets/20771942/2f00129d-f61e-4405-b43f-f5683f4396a0">

So the default behavior on the main branch is already improved a lot over the last release. I can't figure out why that is, the CSS that applies to the `v_list-item` element seems to be exactly the same in both versions.

**Comment by RussellNS** [comment]
Out of curiosity, I’m running version 2.6.0, and I’m not seeing fractions as an option for the ‘quantity’ field.  Am I doing something wrong?  I’d really like to be able to use fractions as a value for quantity.

**Reply by michael-genson** [reply]
The quantity is stored internally as a float, but the unit determines whether it's displayed as a fraction or not:
![image](https://github.com/user-attachments/assets/c3137ebf-1f9c-4358-9f07-e6739e42683a)
